cmd/cape/cmd/list.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"github.com/capeprivacy/cli/entities"
+)
+
+// listCmd represents the run command
+var listCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List all deployed functions",
+	Long:  "List all deployed function, for your user.\n",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := list(cmd, args)
+		if _, ok := err.(UserError); !ok {
+			cmd.SilenceUsage = true
+		}
+		return err
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(listCmd)
+}
+
+func list(cmd *cobra.Command, args []string) error {
+	u := C.SupervisorHost
+	insecure := C.Insecure
+
+	if len(args) > 0 {
+		return UserError{Msg: "too many arguments", Err: fmt.Errorf("invalid number of input arguments")}
```

Comment:
can we make the message more explicit? something like "list does not take in any arguments"